CentralManagement/ChocoCCM Command Reference/HelpersReference.md

```diff
@@ -0,0 +1,383 @@
+
+ * [[Add-CCMGroup|HelpersAddCCMGroup]]
+ * [[Add-CCMGroupMember|HelpersAddCCMGroupMember]]
+ * [[Connect-CCMServer|HelpersConnectCCMServer]]
+ * [[Export-CCMDeployment|HelpersExportCCMDeployment]]
+ * [[Export-CCMDeploymentDetail|HelpersExportCCMDeploymentDetail]]

```

Comment:
Please re-check this file, it looks like the commands were duplicated several times over for some reason.